common/cpp/src/google_smart_card_common/messaging/typed_message_router.cc

```diff
@@ -50,16 +51,12 @@ void TypedMessageRouter::RemoveRoute(TypedMessageListener* listener) {
 
 bool TypedMessageRouter::OnMessageReceived(Value message,
                                            std::string* error_message) {
-  // TODO: Remove this conversion and use `Value` directly.
-  const pp::Var pp_message = ConvertValueToPpVar(message);
-
   TypedMessage typed_message;
-  // TODO: Delete `local_error_message` in favor of (optional) `error_message`.
```

Comment:
As per offline discussion, I've changed it back to `local_error_message`, so that the error message creation code is simpler to understand.